pkgs/development/tools/build-managers/cmake/2.8.nix

```diff
@@ -56,24 +71,24 @@ stdenv.mkDerivation rec {
     (concatMap (p: [ (p.dev or p) (p.out or p) ]) buildInputs);
 
   configureFlags = [
-    "--docdir=/share/doc/${name}"
+    "--docdir=/share/doc/${pname}"
```

Comment:
Shouldn't be `${pname}${version}` instead :thinking: ?

Reply:
totally correct